### PR TITLE
feat2: added archive, delete and colorPicker

### PR DESCRIPTION
**Pull request by SinchanaMS**
Added the Archive, Delete and Color Picker functionalities.

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Coding is looking good. You can merge. 
One thing I noticed is in the app, where we have to write note text. You can increase that area for 3-4 rows. I found it little inconvenient to select to that particular area to write !!

**Review by iHarryD — APPROVED** [review]
- Variable naming is extremely good.
- Code is clean and easy to understand.
- You can use double shots with && wherever you used .length with ternary.

File: src/components/AddNote.jsx [hunk 1]
```diff
@@ -20,7 +30,10 @@ export default function AddNote() {
     } catch (error) {
       console.log(error)
     }
+  } else {
+    Toast({type: "error", message:"Please login"})
```
Comment (bhtibrewal) [inline src/components/AddNote.jsx:34]: Suggestion: name it better something like showToast / addToast
also you should show a toast after updating note in line 26

File: src/components/ArchiveNote.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+export default function ({note}) {
+
+  const {trash, setTrash, setNotes, setArchives} = useNote()
+
+const restoreFromArchive = async (note) =>{
+  try {
+    const response = await axios.post(`/api/archives/restore/${note._id}`, {}, {
+      headers: {
+        authorization: localStorage.getItem("userToken")
```
Comment (bhtibrewal) [inline src/components/ArchiveNote.jsx:13]: Suggestion: store the token in context then you would not have to get it again n again from local

File: src/components/Note.jsx [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+    })
+    if (response.status === 201){
+      setNotes(response.data.notes)
+      setArchives(response.data.archives)
+      Toast({type: "success", message: "Note archived"})
+    }
+  } catch (error) {
+    console.log(error)
+  }
+}
```
Comment (bhtibrewal) [inline src/components/Note.jsx:41]: Suggestion: move this to a seperate file

File: src/components/Note.jsx [hunk 4]
```diff
@@ -1,12 +1,57 @@
...
-    <div className='note'>
+    <div className='note' style={{ backgroundColor: note.bgColor }}>
+      <div className="note-header">
         <p className='note-title'>{note.title}</p>
+      </div>
         <p className='note-body' dangerouslySetInnerHTML={{__html: note.body}}></p>
         <div className='note-footer'>
-            <p>13/02/2022</p>
-            <button className="delete-note"><span class="material-icons material-icons-outlined delete-icon">delete</span></button>
+          <p className="created-date">{note.date}</p>
```
Comment (bhtibrewal) [inline src/components/Note.jsx:50]: Destructure note a top so that you don't have to write note. again n again

**Review by bhtibrewal — APPROVED** [review]
good work the code is clean and readable, written few suggestions and you are good to merge

File: src/components/ArchiveNote.jsx (hunk 2 above)
Comment (aiman-ansari) [reply src/components/ArchiveNote.jsx:13]: Agree with bhavika you can store token in context and use it anywhere

**Comment by SinchanaMS** [comment]
Thank you for the reviews. The suggestions will be addressed in the next refactoring. Merging to dev.

**Review by aiman-ansari — APPROVED** [review]
Naming is good and understandable good folder structure only thing is you can move  all the api call part in different file. 
Otherwise everything is great
Do check suggestion are mentioned

**Pull request merged**